### DISCUSSION TITLE
Table行内编辑请教

### DISCUSSION THREAD

**Discussion by chonglaimuyou** [dotnetcore/BootstrapBlazor#1932] [Q&A] [answered]
当EditMode="EditMode.InCell"，Table行内编辑如何多行同时进入编辑状态？即可以同时编辑多行，然后一次性提交。
例如：
![88](https://github.com/dotnetcore/BootstrapBlazor/assets/40379003/579899be-ff6c-4a06-bb15-433050d2359b)

**Comment by ArgoZhang** [comment, marked as answer]
@chonglaimuyou 目前不支持这样的操作，可以试试看 Excel 模式
